tests/integration/components/speaker-item-test.js

```diff
@@ -0,0 +1,20 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('speaker-item', 'Integration | Component | speaker item');
+
+test('it renders', function(assert) {
+
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });
+
+  // Template block usage:
+  this.render(hbs`
```

Comment:
Speaker-item doesn't use block rendering. Use only `{{speaker-item}}` for the test.